controllers/auth.js

```diff
@@ -0,0 +1,86 @@
+const jwt = require( 'jwt-simple' );
+
+const User = require( '../models/schemas/user' );
+const config = require( '../models/config' );
+
+exports.loginUser = ( req, res, next ) => {
+    if ( typeof req.body.email !== 'string' )

```

Comment:
It can still be `string` and be empty. Do a stricter check by checking length of the input, and validate that's an actual email. Same for `password`.